test/tests/val-2a/comparison_val-2a.py

```diff
@@ -49,7 +48,7 @@ def numerical_solution_on_experiment_input(experiment_input, tmap_input, tmap_ou
 experiment_flux_TMAP4 = experiment_TMAP4_data['permeation flux (atom/m^2/s)']
 
 TMAP4_file_base = 'val-2a_comparison'
-############################ TMAP4 atom/m$^2$/s ############################
+############################ recommendation flux - atom/m$^2$/s ############################
```

Comment:
```suggestion
############################ recombination flux - atom/m$^2$/s ############################
```
Recombination, not recommendation - you keep getting these two confused. :-)